### PR TITLE
Tooltips for three standard products

### PR DESCRIPTION
**Pull request by rfrancel**
I put this as work in progress since the price service is not configured yet on puzzle's side, and also because I think we should look if what I did is compatible with your cutting service @loicgasser 
The three layers concerned are:
ch.swisstopo.swissbuildings3d-2.metadata
ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung
ch.swisstopo.images-spot-5.metadata

### CODE REVIEW

**Comment by rfrancel** [comment]
the three layers are ready to go on prod and the problem is fixed concerning the swissimage double tooltip: [test link](https://mf-geoadmin3.dev.bgdi.ch/frr_standard_products_tooltips/?api_url=ltfrr&topic=ech&lang=fr&X=190000.00&Y=660000.00&zoom=1&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.swissbuildings3d-2.metadata,ch.swisstopo.images-spot-5.metadata,ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung,ch.swisstopo.swissimage-product&layers_opacity=0.75,0.75,1,1&layers_visibility=false,false,false,true)

**Comment by p1d1d1** [comment]
@rfrancel test link does not work

**Comment by rfrancel** [comment]
it should work now @p1d1d1

**Comment by p1d1d1** [comment]
@rfrancel seems to me they are not ready on the Shop side, are they?

**Comment by rfrancel** [comment]
No they are not

**Comment by loicgasser** [comment]
@rfrancel please add missing legends or change BOD related field.

File: chsdi/models/vector/stopo.py [hunk 1]
```diff
@@ -2394,30 +2397,46 @@ class SwissAlti3dPerimeter(Base, ShopStandardClass, Vector):
...
     __totalArea__ = 58500.0
     id = Column('min', Integer, primary_key=True)
     the_geom = Column(Geometry2D)
 
-register('ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung', DHM25Perimeter)
+register('ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung', DHM25)
 register_perimeter('ch.swisstopo.digitales-hoehenmodell_25_reliefschattierung', DHM25Perimeter)
 
 
+class spotMosaicPerimeter(Base, ShopStandardClass, Vector):
```
Comment (loicgasser) [inline chsdi/models/vector/stopo.py]: Spot

**Comment by loicgasser** [comment]
We're missing lkname for ch.swisstopo.pixelkarte-pk200.metadata in DB...
Not your fault, but I can't find the original data.

**Comment by p1d1d1** [comment]
@gjn @rfrancel what about using the "menu" insted of the "button" for ch.swisstopo.swissbuildings3d-2.metadata?
So that one can at least see the name of the commune he's buying?

**Comment by oterral** [comment]
That makes sense for me.

**Comment by loicgasser** [comment]
DB issue fixed

**Comment by rfrancel** [comment]
jenkins test

**Pull request merged**